### PR TITLE
Update protobuf to envoy 1.16.0

### PR DESCRIPTION
**Pull request by wookieJ**

### CODE REVIEW

File: server/src/test/java/io/envoyproxy/controlplane/server/EnvoyContainer.java [hunk 1]
```diff
@@ -24,7 +24,7 @@
   private final Supplier<Integer> controlPlanePortSupplier;
 
   EnvoyContainer(String config, Supplier<Integer> controlPlanePortSupplier) {
-    super("envoyproxy/envoy-alpine-dev:latest");
+    super("envoyproxy/envoy-alpine-dev:8fb3cb86082b17144a80402f5367ae65f06083bd");
```
Comment (slonka) [inline server/src/test/java/io/envoyproxy/controlplane/server/EnvoyContainer.java:27]: @wookieJ - please add that this needs to be updated here: https://github.com/envoyproxy/java-control-plane#bring-api-up-to-date-with-data-plane-api. Also @nezdolik I think that this is a better way to have repeatable builds and solves this https://github.com/envoyproxy/java-control-plane/issues/154 . In the next release of java-control-plane we will remove v2 usage and update to to 1.16.1.

**Review by slonka — CHANGES_REQUESTED** [review]
Have you manually touched any files other than `tools/update-api.sh`, `tools/API_SHAS`, `EnvoyContainer.java` ? The diff is pretty big and quite hard to navigate.

File: tools/API_SHAS [hunk 2]
```diff
@@ -1,11 +1,11 @@
 # Update the versions here and run update-api.sh
 
 # envoy (source: SHA from https://github.com/envoyproxy/envoy)
-ENVOY_SHA="38fe5e1ec115c0b2e83921211c2646f160a91681" # Feb 13, 2020
+ENVOY_SHA="8fb3cb86082b17144a80402f5367ae65f06083bd"  # Oct 8, 2020
```
Comment (MarcinFalkowski) [inline tools/API_SHAS:4]: Maybe it is worth to add `(v1.16.0)` to the comment?

File: server/src/test/java/io/envoyproxy/controlplane/server/EnvoyContainer.java (hunk 1 above)
Comment (wookieJ) [reply server/src/test/java/io/envoyproxy/controlplane/server/EnvoyContainer.java:27]: no, manual changes only to the files you specified. And `README.md` soon

File: tools/API_SHAS (hunk 2 above)
Comment (wookieJ) [reply tools/API_SHAS]: added

File: server/src/test/java/io/envoyproxy/controlplane/server/EnvoyContainer.java (hunk 1 above)
Comment (wookieJ) [reply server/src/test/java/io/envoyproxy/controlplane/server/EnvoyContainer.java:27]: added

**Review by MarcinFalkowski — APPROVED** [review]

File: server/src/test/java/io/envoyproxy/controlplane/server/EnvoyContainer.java (hunk 1 above)
Comment (nezdolik) [reply server/src/test/java/io/envoyproxy/controlplane/server/EnvoyContainer.java:27]: nice! lmk if I could help with deprecating v2

**Review by slonka — APPROVED** [review]

**Pull request merged**